static/app/views/replays/list/replaySearchAlert.tsx

```diff
@@ -1,133 +1,137 @@
   const location = useLocation();
-  const organization = useOrganization();
-  const sdkUpdates = useProjectSdkUpdates({
-    organization,
-    projectId: null,
-  });
-
-  const {dismiss: handleDismiss, isDismissed} = useDismissAlert({
-    key: LOCAL_STORAGE_KEY,
+  const {dismiss, isDismissed} = useDismissAlert({
+    key: REPLAY_CLICK_SEARCH_FEATURE_BANNER_KEY,
   });
-  const conditions = useMemo(() => {
-    return new MutableSearch(decodeScalar(location.query.query, ''));
-  }, [location.query.query]);
 
-  const hasReplayClick = conditions.getFilterKeys().some(k => k.startsWith('click.'));
-
-  if (sdkUpdates.type !== 'resolved') {
+  if (isDismissed) {
     return null;
   }
+  const heading = (
+    <Fragment>
+      {tct('Introducing [feature]', {
+        feature: (
+          <ExternalLink href="https://blog.sentry.io/introducing-search-by-user-click-for-session-replay-zero-in-on-interesting/">
+            {t('Click Search')}
+          </ExternalLink>
+        ),
+      })}
+    </Fragment>
+  );
 
-  const selectedProjectsWithSdkUpdates = sdkUpdates.data?.reduce((acc, sdkUpdate) => {
-    if (!selection.projects.includes(Number(sdkUpdate.projectId))) {
-      return acc;
-    }
-
-    const project = projects.projects.find(p => p.id === sdkUpdate.projectId);
-    // should never really happen but making ts happy
-    if (!project) {
-      return acc;
-    }
+  const description = (
+    <span>
+      {tct(
+        `Find replays which captured specific DOM elements using our new search key [key]`,
+        {
+          key: <strong>{t("'click'")}</strong>,
+        }
+      )}
+    </span>
+  );
 
-    acc.push({
-      project,
-      sdkUpdate,
+  const handleTryNow = () => {
+    browserHistory.push({
+      ...location,
+      query: {
+        ...location.query,
+        query: 'click.tag:button',
+      },
     });
+    dismiss();
+  };
 
-    return acc;
-  }, [] as Array<{project: Project; sdkUpdate: ProjectSdkUpdates}>);
+  const handleLearnMore = () => {
+    openModal(LearnMoreModal);
+  };
 
-  const doesNotMeetMinSDK =
-    selectedProjectsWithSdkUpdates &&
-    selectedProjectsWithSdkUpdates.length > 0 &&
-    selectedProjectsWithSdkUpdates.every(({sdkUpdate}) => {
-      return semverCompare(sdkUpdate.sdkVersion, MIN_REPLAY_CLICK_SDK) === -1;
-    });
-
-  if (!doesNotMeetMinSDK) {
+  if (isDismissed) {
     return null;
   }
 
-  if (hasReplayClick) {
+  if (needSdkUpdates) {
     return (
-      <Alert data-test-id="min-sdk-alert">
-        <AlertContent>
-          <IconInfo />
-          <AlertText>
-            {tct(
-              'Search field [click] requires a minimum SDK version of >= [minSdkVersion].',
-              {
-                click: <strong>'click'</strong>,
-                minSdkVersion: <strong>{MIN_REPLAY_CLICK_SDK}</strong>,
-              }
-            )}
-          </AlertText>
-        </AlertContent>
-      </Alert>
+      <Fragment>
+        <NewFeatureBanner
```

Comment:
this is annoying, but should we have this inside getsentry, or behind a feature flag?

Reply:
What do we do for the AI banner in issue details? We should follow that. Since this is not billing related, it might be fine to have in sentry. It's a prompt to get users to upgrade their SDK, which would apply to self hosted.